dev-ros/rosbridge_server/rosbridge_server-0.11.10.ebuild

```diff
@@ -20,7 +20,7 @@ RDEPEND="
 	dev-ros/rospy[${PYTHON_SINGLE_USEDEP}]
 	dev-ros/rosauth[${PYTHON_SINGLE_USEDEP}]
 
-	$(python_gen_cond_dep "www-servers/tornado[\${PYTHON_USEDEP}]")
+	$(python_gen_cond_dep "dev-python/tornado[\${PYTHON_USEDEP}]")
 	$(python_gen_cond_dep "dev-python/autobahn[\${PYTHON_USEDEP}]")
 	$(python_gen_cond_dep "dev-python/twisted[\${PYTHON_USEDEP}]")
```

Comment:
I know it's not your fault but could you combine this shit into one call, while at it?